packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -105,6 +112,13 @@ class CpuProfileData {
       parent.addChild(stackFrame);
     }
   }
+
+  void _setLeafCounts() {
+    for (Map<String, dynamic> traceEvent in stackTraceEvents) {
+      final leafId = traceEvent['sf'];
```

Comment:
make all this field names like 'sf'
constants so you can add comments describing what each one is so this code is simpler to review. As is, I have no idea what the 'sf' field is.

Reply:
done.